be/src/Infrastructure/Catalog/Foundation.Catalog/Infrastructure/Products/Entities/Product.cs

```diff
@@ -31,6 +31,8 @@ public class Product : Entity
 
         public string Ean { get; set; }
 
+        public int FulfillmentTime { get; set; }
```

Comment:
**suggestion** It should be nullable if it isn't required. If you change please generate new migration